client/util.js

```diff
@@ -55,8 +55,9 @@ window.AudioContext = window.AudioContext || window.webkitAudioContext;
 			});
 
 			// Set the scale for OBJ models based on the known width of the table model.
-			var tableModelWidth = 297.444;
+			var tableModelWidth = 297.444, tableModelHeight = 103;
 			var objScale = 1 / tableModelWidth * tableRadius * 2;
+			var objVScale = 0.8 / tableModelHeight;
```

Comment:
Maybe name this `tableRelativeVerticalScale` or something. `objVScale` is not very descriptive.